ts/session/apis/snode_api/retrieveRequest.ts

```diff
@@ -167,11 +167,18 @@ async function retrieveNextMessages(
     GetNetworkTime.handleTimestampOffsetFromNetwork('retrieve', bodyFirstResult.t);
 
     // merge results with their corresponding namespaces
-    return results.map((result, index) => ({
-      code: result.code,
-      messages: result.body as RetrieveMessagesResultsContent,
-      namespace: namespaces[index],
-    }));
+    return results.map((result, index) => {
+      const messages = result.body as RetrieveMessagesResultsContent;
+      // Not sure if that makes sense, but we probably want those messages sorted.
+      const sortedMessages = sortBy(messages.messages, m => m.timestamp);
```

Comment:
I guess if we can't trust the order we receive the messages from the server then this does make sense? 🙏